### ISSUE TITLE
Override jump to keyframe (⬆⬇) to also jump to cut, including during playback

### ISSUE THREAD

**Issue by NathanLovato** [GDquest/Blender-power-sequencer#220]
The idea is to make it so the ⬆ and ⬇ jump to the previous and next cut, reliably (during playback, which blender's default tool doesn't support)

Optional idea: automatically select linked strips when moving to cut?

**Comment by 3pointedit** [comment]
Arrow up/Down is jump to keyframe though, or do you mean page up and page down?

**Comment by NathanLovato** [comment]
From the title:
```md
Override jump to keyframe (⬆⬇) to **also** jump to cut, including during playback
```

**Comment by 3pointedit** [comment]
Oops sorry. It seems like all your replies to me are "from the title".
Really very sorry

**Comment by NathanLovato** [comment]
No worries!

**Comment by NathanLovato** [comment]
My bad, closed the wrong issue when comitting

**Comment by henriiquecampos** [comment]
Hey I just started trying to implement this, and I figured out that we already have an operator for that, [jump_to_cut](https://github.com/GDquest/Blender-power-sequencer/blob/master/operators/jump_to_cut.py), so the issue is about making so when we press :arrow_up: or :arrow_down: we go to whatever comes first: cut or keyframe? And also make it so the jump to keyframe happens during playback. Does that sound correct?
